src/one_dragon_qt/widgets/setting_card/value_display_editable_combo_box_setting_card.py

```
@@ -0,0 +1,190 @@
+from PySide6.QtGui import QIcon
+from qfluentwidgets import CaptionLabel, FluentIconBase
+
+from one_dragon.base.config.config_item import ConfigItem
+from one_dragon_qt.utils.layout_utils import IconSize, Margins
+from one_dragon_qt.widgets.setting_card.editable_combo_box_setting_card import (
+    EditableComboBoxSettingCard,
+)
+
+
+class ValueDisplayEditableComboBoxSettingCard(EditableComboBoxSettingCard):
+    """可输入真实值，并在右侧显示匹配选项外显名的下拉设置卡片。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)